### DISCUSSION TITLE
Zoom or zoom features on mobile devices.

### DISCUSSION THREAD

**Discussion by nloshka404** [firefly-iii/firefly-iii#7490] [General]
Please tell me the zoom functions or otherwise the zoom functions stopped working on a mobile device. sometimes you need to enlarge part of the screen for a more detailed view or screenshot. But with the release of new releases, the zoom functions stopped working.
Are you planning to bring zoom back to mobile devices?
It is sometimes very difficult to work without these functions. The font is small and very hard to read until you enlarge it.

**Comment by JC5** [comment]
Mmmmm, I'm not aware of any changes. Which OS l, device and browser?

**Reply by nloshka404** [reply]
A regular Samsung phone on an android browser is either chrome or regular

**Reply by JC5** [reply]
In Chrome, make sure you set the "force zoom" option on.

**Reply by nloshka404** [reply]
oh, I'm a mess. I turned on the option in Google to force scale and everything worked :-) Thank you.

**Reply by JC5** [reply]
Not a problem, glad it works!
